docs/applications/link-server-to-application.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Link a Server to an Application
+intro: Link a server to an application to select which servers an application can be deployed.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/configuration-files/add-configuration-file
+    next:
+        - /docs/applications/deploy-application
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DETAILS` on the application you want to link a server
+1. On the upper-right corner of the application details view, click `...`
+1. Use the drop-down menu to choose `Linked resources`
+1. On the upper-right corner of the `Linked resources` list, click `LINK TO A SERVER`
+1. Follow the prompts then click `Link`
+    - The selected server will be listed in the `DEPENDS ON` table on the `Linked resources` list
```

Comment:
`Follow the prompts ...` + `the selected ...` is really the pattern we've been following in other topics that use select/drop down elements? 🤔 
`Follow the prompts` was usually adopted (by you) when there were more than one thing/element to interact with.

Please re-consider which approach is the one you recommend and let's stick to a single one.

Reply:
I agree, it's a pattern change.

Added a new step to the list, could you please re-review?

Reply:
Applied the same pattern used in `docs/cloud-provider-credentials/add-aws-credential.md`

```md
...
1. Follow the prompts then click `Create user`
1. In the `Users` list, click the username link of the user recently created
...
```

Reply:
@jfoliveira could you review if the applied changes solves the thread?